packages/core/models/translation-object.model.spec.ts

```diff
@@ -6,4 +6,10 @@ describe("TranslationObject", () => {
         expect(obj).toBeDefined();
         expect(obj.get("en")).toEqual("Hello World!");
     });
+
+    it("Setting disableFallback at true should return not fallback the to the first available translation", () => {
+        const obj = new TranslationObject({ fr: "Test!" }, { disableFallback: true });
+        expect(obj).toBeDefined();
+        expect(obj.getOrFirstIfNull("en")).toEqual(null);
```

Comment:
On devrait aussi "fr"
```suggestion
        expect(obj.getOrFirstIfNull("en")).toEqual(null);
        expect(obj.getOrFirstIfNull("fr")).toEqual("Test!");
```